build/cmake/DownloadSoundFont.cmake

```diff
@@ -2,7 +2,7 @@
 
 set (SF_VERSION_REMOTE_FILE ${PROJECT_BINARY_DIR}/SF_VERSION)
 set (SF_VERSION_LOCAL_FILE ${PROJECT_SOURCE_DIR}/share/sound/SF_VERSION)
-set (DOWNLOAD_URL_ROOT "https://ftp.osuosl.org/pub/musescore/soundfont/MuseScore_General")
+set (DOWNLOAD_URL_ROOT "https://ftp.osuosl.org/pub/musescore/soundfont/MS Basic")
```

Comment:
I recommend moving to S3, this ftp was used for historical reasons when there was no S3

Reply:
moved